lib/executables/validate.js

```diff
@@ -0,0 +1,146 @@
+/* eslint-disable no-use-before-define */
```

Comment:
There are some additional checks done by gitReceiver when pushed to the platform: https://github.com/elasticio/elasticio/blob/f51712276c60a75ad39c52f6ba00bea22c2f38e2/src/gitreceiver/src/lib/metaloader.js  It may be logical to try to duplicate/extend those.

Reply:
Just for my own reference to look at this tomorrow, these are:

- assureFuncsValidNaming
- assureFuncsUniqNaming